### ISSUE TITLE
[Feature request] Customizable hotkeys 

### ISSUE THREAD

**Issue by xan2622** [dividebysandwich/F117A-remake#1]
Hi. 

I have noticed that you added "☑️ **Basic controls**" in your todo list.

The WASD hotkeys are compatible with the [QWERTY](https://en.wikipedia.org/wiki/QWERTY) and the [QWERTZ](https://en.wikipedia.org/wiki/QWERTZ) keyboard layouts, but not with [AZERTY](https://en.wikipedia.org/wiki/AZERTY).

For example, the french keyboards use the AZERTY keyboard layout and the WASD hotkeys are not really easy to use (ZQSD is used instead). 

When your game is more complete, could you please consider adding an option to let the user (the player) customize hotkeys? 

Thanks in advance.

**Comment by dividebysandwich** [comment]
Good point! I shall incorporate that via bevy_input_mapper at some point.
